lib/module.js

```diff
@@ -51,6 +51,11 @@ module.exports = async function gtmModule (_options) {
   script = `if(!window._gtm_init){window._gtm_init=1;${script}}`
 
   // Add google tag manager <script> to head
+  if (typeof this.options.head === 'function') {
+    // eslint-disable-next-line no-console
+    console.warn('[@nuxtjs/gtm] head is provided as a function which is not supported by this module at the moment. Removing user-provided head.')
+    this.options.head = typeof this.options.head === 'function' ? {} : this.options.head
```

Comment:
I think this `typeof` check is not necessary anymore (upper-level condition is checking)

Reply:
i'm fixing.. T0T

Reply:
updated

Reply:
am i try to first PR of my develop life.. please understand to insufficient processes.

Reply:
NP! Happy to hear you are willing to do more OSS stuff :) Wish to see more contributions by you :+1:

Reply:
i'm glad you, i will try to fix and improve more OSS! 
thanks